src/lib/Functions/handlePaste.ts

```diff
@@ -4,6 +4,18 @@ import { ClipboardEvent } from '../Model/domEventsTypes';
 import { getActiveSelectedRange } from './getActiveSelectedRange';
 import { pasteData } from './pasteData';
 
+const decimalSeparator = (1.1).toLocaleString().substring(1, 2);
+const thousandSeparator = (1000).toLocaleString().substring(1, 2);
+
+function parseFloatLocale(text: string): number {
+    return parseFloat(
+      text
+        .replace(thousandSeparator, "")
+        .replace(decimalSeparator, ".")
+        .replace(/[^0-9.-]/g, "")
+    );
+}
```

Comment:
I think it can be simplified to just:
```suggestion
const localeParseFloat(text: string): number  => parseFloat(text.replaceAll(".","").replaceAll(",","."));
```